src/global/helpers/position.ts

```diff
@@ -1,3 +1,5 @@
+export type Alignment = { horizontal: 'start' | 'center' | 'end'; vertical: 'above' | 'below' };
```

Comment:
Can you change that to an interface?
```suggestion
export interface Alignment = { horizontal: 'start' | 'center' | 'end'; vertical: 'above' | 'below' };
```